targets/targets.json

```diff
@@ -8627,7 +8627,8 @@
         "inherits": ["GD32_Target"],
         "supported_form_factors": ["ARDUINO"],
         "core": "Cortex-M4F",
-        "extra_labels_add": ["GD32F4XX", "GD32F450ZI", "GD_EMAC"],
+        "components_add": ["FLASHIAP"],
```

Comment:
is this correct ? flashiap is not a component, rather should be in `device_has`

Reply:
> 
> 
> is this correct ? flashiap is not a component, rather should be in `device_has`

we have this storage component in cloud client example. ..\mbed-os\components\storage\blockdevice\COMPONENT_FLASHIAP

Reply:
+1